py/torch_tensorrt/fx/fx2trt.py

```diff
@@ -153,7 +153,6 @@ def validate_conversion(self):
 
     def run(
         self,
-        max_batch_size=64,
```

Comment:
I am afraid we can not do this change. We have to maintain backward compatibility on the API, otherwise, it will break our internal product.